common/test/acceptance/pages/studio/utils.py

```diff
@@ -101,6 +101,16 @@ def add_component(page, item_type, specific_type, is_advanced_problem=False):
     page.wait_for_ajax()
 
 
+def add_components(page, item_type, items, is_advanced_problem=False):
+    """
+    Adds multiple components of a specific type.
+    item_type should be "advanced", "html", "problem", or "video"
+    items is a list of components of specific type to be added.
+    """
```

Comment:
This implementation has a limitation that the value of "is_advanced_problem" will apply to all the items in the list.
E.g. you won't be able to create in one call both a "Blank Common Problem" and a "Blank Advanced Problem" because those two would need opposite values for is_advanced_problem.

Couple choices:
* Leave this as-is, but document that in a single call if you want to create an advanced problem, all of the other items in the list that you are creating must also be advanced problems.
* Change the method to have arguments page, item_type, and items where items is a list of tuples. The tuple will be a string (the item type) and a boolean (is this an advanced problem). What is potentially confusing is that the boolean should also be set as false for any non-problem type (e.g. advanced, html, video).

I'm good with either solution.